downloads/kubernetes/quick-install.yml

```diff
@@ -230,7 +230,7 @@ data:
         apiSecurity:
           ssl:
             enabled: false
-          overrideBaseUrl: http://localhost:9000/gate
+          overrideBaseUrl: /gate
```

Comment:
Does this work when you're not running on cloud shell?

Reply:
yes